------------------------------------------------------------------------
r84049 | ripley | 2023-03-25 10:01:30 +0100 (sob, 25 mar 2023) | 1 line

allow libcurl 8 as its API/ABI is said to be unchanged

Index: configure
===================================================================
--- configure	(revision 84048)
+++ configure	(revision 84049)
@@ -51680,8 +51680,8 @@
 done
 
 if test "x${have_libcurl}" = "xyes"; then
-{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if libcurl is version 7 and >= 7.28.0" >&5
-printf %s "checking if libcurl is version 7 and >= 7.28.0... " >&6; }
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if libcurl is >= 7.28.0" >&5
+printf %s "checking if libcurl is >= 7.28.0... " >&6; }
 if test ${r_cv_have_curl728+y}
 then :
   printf %s "(cached) " >&6
@@ -51699,7 +51699,7 @@
 {
 #ifdef LIBCURL_VERSION_MAJOR
 #if LIBCURL_VERSION_MAJOR > 7
-  exit(1);
+  exit(0);
 #elif LIBCURL_VERSION_MAJOR == 7 && LIBCURL_VERSION_MINOR >= 28
   exit(0);
 #else
Index: m4/R.m4
===================================================================
--- m4/R.m4	(revision 84048)
+++ m4/R.m4	(revision 84049)
@@ -4620,7 +4620,7 @@
 AC_CHECK_HEADERS(curl/curl.h, [have_libcurl=yes], [have_libcurl=no])
 
 if test "x${have_libcurl}" = "xyes"; then
-AC_CACHE_CHECK([if libcurl is version 7 and >= 7.28.0], [r_cv_have_curl728],
+AC_CACHE_CHECK([if libcurl is >= 7.28.0], [r_cv_have_curl728],
 [AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <stdlib.h>
 #include <curl/curl.h>
@@ -4628,7 +4628,7 @@
 {
 #ifdef LIBCURL_VERSION_MAJOR
 #if LIBCURL_VERSION_MAJOR > 7
-  exit(1);
+  exit(0);
 #elif LIBCURL_VERSION_MAJOR == 7 && LIBCURL_VERSION_MINOR >= 28
   exit(0);
 #else

------------------------------------------------------------------------
